src/main/resources/application.yml

```diff
@@ -1,3 +1,12 @@
+management.metrics.enable:
+    all: false
+    application: true # reports startup time, useful for future comparison
+    http.server.requests: true # reports http request counts
+    resilience4j.circuitbreaker: true # reports circuit breaker status
+    jvm.memory: true # reports heap status
+    jvm.gc: true # reports garbage collector metrics, useful to detect too long pauses
+    mongodb.driver: true # reports useful mongo statistics, like waiting queue
```

Comment:
I would expose only requests count metrics disabling others. those can be enabled in the future as needed. 
What do you think?

Reply:
Yes, I'm agree, the purpose was to get more data during testing, so now, I will disable useless metrics :)